test/integration/controllers/users_controller_test.js

```diff
@@ -1066,7 +1066,40 @@ describe('Endpoint tests', () => {
       });
     });
 
-    it('removes a recipe from breakfast successfully', (done) => {
+    it('removes all recipes from breakfast successfully', (done) => {
```

Comment:
How is this test different from the one that works for some of the recipes, if it works for let say 2 out f 5, shouldn't it work for 5 out of 5 too. Unless there is a major difference in process

Reply:
there is yeah, this was where the bug occured